includes/class-wc-payments-features.php

```diff
@@ -226,6 +226,15 @@ public static function is_progressive_onboarding_enabled(): bool {
 		return '1' === get_option( self::PROGRESSIVE_ONBOARDING_FLAG_NAME, '0' );
 	}
 
+	/**
+	 * Checks whether the Fraud Protection settings section display flag is enabled.
```

Comment:
We could remove the "section display" part from this comment as this feature flag will be used for everything related to the fraud protection settings.

Reply:
Thanks for catching this! I've updated the doc block in [072337b](https://github.com/Automattic/woocommerce-payments/pull/5645/commits/072337b329cdee1d458609a33f62c55dd1c3e818).